index.html

```diff
@@ -1,52 +1,46 @@
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
-    <title>Portfolio</title>
-</head>
-<body>
-  <section class="first-page">
-    <div class="main">
-      <nav class="navbar">
-        <div class="logo"><a href="#">Eros</a></div>
-        <div class="menu"><i class='bx bx-menu'></i></box-icon></div>
-    </nav>
-    <div class="headline">
-    <div class="description">
-      <h1 class="name"> I am Aime
-          Glad to see you!</h1>
-         
-      <p class="text"> I am a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-           </p>
-      </div>
-      <div class="contact">
-        <button>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css"rel="stylesheet"/>
+    <link rel="stylesheet" href="styles.css"/>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap"rel="stylesheet"/>
 
-          <h2 class="media">LET'S CONNECT:</h2>
-        
-        </button>
-        <ul>
-         
-          <li><a href="https://twitter.com/AimeSebagabo"><i class='bx bxl-twitter'></i></a></li>
+    <title>Porfolio</title>
+  </head>
+  <body>
+    <header class="mobile-portfolio">
+      <a href="#" class="logo">
+        <h2>Eros</h2>
+      </a>
+      <nav class="menu"><img src="./image/Union.svg" alt=""></nav>
+    </header>
+    <section class="first-page">
+      <h2 class="name">I'm Aime <br>
+        Glad to see you!</h2>
 
-          <li><a href="https://www.linkedin.com/in/jean-aime-sebagabo-279aa2207/"></a><i class='bx bxl-linkedin'></i></a></li>
-      
-           <li><a href="http://yourgmailhandle"><i class='bx bxl-gmail'></i></a></li>
-      
-           <li><a href="https://github.com/Hazard10-eng"><i class='bx bxl-github'></i></a</a</li>
-        </ul>
-        </div>
-    </div>
-    </div>
-  </section>
+      <p class="text">
+        I am a software developer! I can help you build a product, feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="contact">
+        <a href="#" class="connect">LET'S CONNECT</a>
+        <ul class="social-icons">
```

Comment:
- [ ] I see that you changed the style of the link, but  As stated by the previous reviewer ,kindly☺️ try to fix just the margin-left to be like the design in the Figma and fix the style to be like the design in the Figma 💯

| Figma design | Your design| 
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/212351579-721dc2a9-5467-41b1-aa37-f5eabc2680ba.png)   |    ![image](https://user-images.githubusercontent.com/83360636/212351519-94694958-c5ab-485d-8102-183a9056194d.png)|